lib/livebook/settings.ex

```diff
@@ -117,4 +120,160 @@ defmodule Livebook.Settings do
   def set_update_check_enabled(enabled) do
     storage().insert(:settings, "global", update_check_enabled: enabled)
   end
+
+  @doc """
+  Gets a list of environment variables from storage.
+  """
+  @spec fetch_env_vars() :: list(EnvironmentVariable.t())
+  def fetch_env_vars do
+    for fields <- storage().all(:environment_variables) do
+      struct!(EnvironmentVariable, fields)
+    end
+  end
+
+  @doc """
+  Gets one environment variable from storage.
+
+  Raises `RuntimeError` if the environment variable does not exist.
+  """
+  @spec fetch_env_var!(String.t()) :: EnvironmentVariable.t()
+  def fetch_env_var!(id) do
+    case storage().fetch(:environment_variables, id) do
+      :error -> raise RuntimeError, "the environment variable #{id} does not exists in storage"
```

Comment:
We want to raise something that will be shown as NotFound instead. Search for `plug_status: 404`. :)